webpack.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
   },
   module: {
     rules: [
+      { enforce: "pre", test: /\.(js|vue)$/, exclude: /node_modules/, loader: "eslint-loader"},
```

Comment:
Minor: formatting should match the rest of the project. We should probably lint this file too :)